### ISSUE TITLE
Ruby 2.7 Warning: Using the last argument as keyword parameters is deprecated (v6.10.0.364)

### ISSUE THREAD

**Issue by fukayatsu** [newrelic/rpm#334]
I got another Ruby 2.7 keyword arguments warnings like #314.

### To Reproduce 

Save below codes as `newrelic_rpm-ruby2-7.rb` and then run `ruby newrelic_rpm-ruby2-7.rb`

```ruby
#!/usr/bin/env ruby

require 'bundler/inline'

gemfile do
  source 'https://rubygems.org'
  gem 'newrelic_rpm', '6.10.0.364' # latest - April 07, 2020 (354KB)
end

require 'new_relic/agent/method_tracer'

class Foo
  class << self
    include ::NewRelic::Agent::MethodTracer

    def bar(arg1, arg2: true)
      puts arg1, arg2
    end

    add_method_tracer :bar, 'Custom/Foo/bar'
  end
end

Foo.bar('baz', arg2: false)

```

### Expected behavior

output:

```
baz
false
```

### Actual behavior

output:

```
/Users/fukayatsu/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/newrelic_rpm-6.10.0.364/lib/new_relic/agent/method_tracer.rb:330: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
newrelic_rpm-ruby2-7.rb:16: warning: The called method `bar_without_trace_Custom_Foo_bar' is defined here
baz
false
```

### ruby -v

```
ruby 2.7.1p83 (2020-03-31 revision a0c7c23c9c) [x86_64-darwin19]
```

**Comment by rachelleahklein** [comment]
Thanks so much for bringing this to our attention, @fukayatsu! Sorry for our oversight in not addressing this in our earlier Ruby 2.7 support. We plan to eliminate these warnings in our next agent release - we'll keep you posted on when it comes out.

**Comment by tannalynn** [comment]
This has been fixed in our newest version 6.11.0.
Thank you for bringing it to our attention!
